web/packages/design/src/Button/buttons.story.tsx

```diff
@@ -118,7 +118,9 @@ export const Buttons = () => {
       <Button block>block = true</Button>
       <Flex gap={3}>
         <Button disabled>Disabled</Button>
-        <Button autoFocus>Focused</Button>
+        <Box className="teleport-button__force-focus-visible">
```

Comment:
I removed `autoFocus` so that the story doesn't automatically scroll this button into viewport. `teleport-button__force-focus-visible` is a custom class that was written to demonstrate the focus state of buttons, so I used it here instead of `autoFocus` to show the focus state.

Reply:
Ahhh that's why the story was starting there. I was recreating the button in Chakra and thought it was weird that it kept jumping half way down